### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by olya-yevsieienko**
[DEMO LINK](https://olya-yevsieienko.github.io/react_goods-selector/)

### CODE REVIEW

**Review by illavodonis — APPROVED** [review]
cool job :fire:

File: src/App.tsx [hunk 1]
```diff
@@ -14,11 +15,102 @@ const goodsFromServer: string[] = [
...
+    }
+
+    if (this.state.selectedGoods.length === 1) {
+      return `${this.state.selectedGoods} is selected`;
+    }
+
+    return `${this.state.selectedGoods.slice(0, -1).join(', ')} and ${this.state.selectedGoods.slice(-1)} are selected`;
+  }
+
+  addedGoods = (good: string) => {
```
Comment (Olexandr6) [inline src/App.tsx:39]: function name should start with verb

Comment (Olexandr6) [reply src/App.tsx:39]: fix this in all places

**Review by Olexandr6 — APPROVED** [review]
Nicely done, great gob🔥

File: src/App.tsx (hunk 1 above)
Comment (olya-yevsieienko) [reply src/App.tsx:39]: Okay, thanks!

**Pull request closed without merging**